packages/esm-patient-registration-app/src/patient-registration/section/section-wrapper.component.tsx

```diff
@@ -31,6 +39,29 @@ export const SectionWrapper = ({ sectionDefinition, index }: SectionWrapperProps
         {t('allFieldsRequiredText', 'All fields are required unless marked optional')}
       </span>
       <div style={{ margin: '1rem 0 1rem' }}>
+        <Joyride
```

Comment:
Hi @Vijaykv5, Can we implement this part inside `patient-registration.component.tsx` (or maybe in `root.component.tsx`)? The current wrapper component you used is only a section wrapper, not a wrapper for the entire component.

Reply:
Thanks @Piumal1999 Now the rendering get's fixed

Reply:
Thanks @Piumal1999 
The rendering issue got fixed